datalab/train_subset.ipynb

```diff
@@ -0,0 +1,501 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classifier Training on Caltech-256 Subset\n",
+    "\n",
+    "In this notebook, we will train a Swin Transformer model for image classifier on a subset of the Caltech-256 dataset using the Timm library and Pytorch. The main steps include dataset sampling and precrocessing, model training with k-fold cross-validation, and feature extraction.\n"
+   ]
+  },
```

Comment:
clarify somewhere that Datalab does not actually require both features and pred_probs, but providing both allows it to more comprehensively detect issues in a dataset.